runMolecule.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-SCENARIOS="default signature custom-bin-file custom-config-file custom-config-dir"
+SCENARIOS="default signature custom-bin-file custom-config-file custom-config-dir custom-all"
 
-MUST_FAIL="centos6:custom-config-file centos6:custom-config-dir"
+MUST_FAIL="centos6:custom-config-file centos6:custom-config-dir centos6:custom-all"
```

Comment:
I get the feeling there is too much of `must fail`  scenarios. Anyway to fix this to actually make it pass, without getting isane ;-) ?
Or maybe just drop centos6.

Reply:
The point is to be sure that the role correctly fails in that cases so none will be unable to login. So I think it safer to keep it as it is

Reply:
I guess @nvtkaszpir is more or less about touching the point we have been struggling with lately. Regardless how many "must fail" scenarios we want to maintain, we didn't find a sane way to cleanly test a "must fail scenario" within the molecule test framework. I mean, how to write a test that's successful when Ansible complains in applying the role to a specific (incompatible) scenario with a given error message? It would have been nice if Ansible had a builtin mechanism to early fail based on some distro compatibility metadata, but I'm not aware of any. I'm also very concerned about long term maintenance of this `runMolecule.sh`. As @macno pointed out, trying to forcefully apply the role to well known "must fail" scenarios could render the system unaccessible. Then we need to be extra safe here. Suggestions for a better approach here are hugely appreciated!